### DISCUSSION TITLE
表格识别标注单元格时，将单元格标注框放大时，会影响整体表格识别的效果吗？

### DISCUSSION THREAD

**Discussion by 18mingmin** [PaddlePaddle/PaddleOCR#13040] [Q&A]
请问在使用标注软件标注时，如果将单元格标注框放大时，会影响整体表格识别的效果吗？
在标注复杂合并单元格时，将单元格标注框放大可以提高识别合并单元格的效果吗
<img width="274" alt="屏幕截图 2024-01-23 165312" src="https://github.com/PaddlePaddle/PaddleOCR/assets/148733315/98481267-e74e-4f88-a6a1-3c5283509f3b">
如果修改完标注框，不点击重识别，会影响表格结构识别训练效果吗

**Comment by dc6273632** [comment]
同问

**Comment by hao-vizseek** [comment]
同问

**Comment by SWHL** [comment]
这个是否影响识别效果，取决于训练数据集中表格单元格标注是什么样的？和训练集中接近，就会更好
